[2022-03-15 19:33:04,533] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 manual__2022-03-15T19:32:54.743377+00:00 [queued]>
[2022-03-15 19:33:04,638] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 manual__2022-03-15T19:32:54.743377+00:00 [queued]>
[2022-03-15 19:33:04,639] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:33:04,639] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-03-15 19:33:04,639] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:33:04,687] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_3> on 2022-03-15 19:32:54.743377+00:00
[2022-03-15 19:33:04,725] {standard_task_runner.py:52} INFO - Started process 12255 to run task
[2022-03-15 19:33:04,769] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_rule', 'task_3', 'manual__2022-03-15T19:32:54.743377+00:00', '--job-id', '157', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule.py', '--cfg-path', '/tmp/tmpjtiunu3y', '--error-file', '/tmp/tmpar1u3xa9']
[2022-03-15 19:33:04,782] {standard_task_runner.py:77} INFO - Job 157: Subtask task_3
[2022-03-15 19:33:05,851] {logging_mixin.py:109} INFO - Running <TaskInstance: trigger_rule.task_3 manual__2022-03-15T19:32:54.743377+00:00 [running]> on host airflowvm
[2022-03-15 19:33:06,984] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T19:32:54.743377+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-15T19:32:54.743377+00:00
[2022-03-15 19:33:06,987] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-15 19:33:06,987] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'exit 0']
[2022-03-15 19:33:07,150] {subprocess.py:85} INFO - Output:
[2022-03-15 19:33:07,181] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-15 19:33:07,378] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_3, execution_date=20220315T193254, start_date=20220315T193304, end_date=20220315T193307
[2022-03-15 19:33:07,568] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 19:33:07,825] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
